### PR TITLE
add vk project

### PR DESCRIPTION
**Pull request by rbitia**
Adding virtual kubelet as a project for google summer of code!

### CODE REVIEW

File: 2018.md [hunk 1]
```diff
@@ -308,3 +308,10 @@ repo](https://github.com/theupdateframework/tuf).
 * Mentors: [Justin Cappos](https://github.com/JustinCappos),
   [Vladimir Diaz](https://github.com/vladimir.v.diaz), and
   [Sebastien Awwad](https://github.com/awwad)
+
+#### Conformance testing for Virtual Kubelet
```
Comment (nikhita) [inline 2018.md]: @rbitia Can you add a header for Virtual Kubelet like https://github.com/cncf/soc#prometheus or https://github.com/cncf/soc#open-policy-agent-opa? Thanks!

**Comment by palash25** [comment]
shouldn't this year's project ideas be added to README.md instead of 2018.md ?

**Comment by caniszczyk** [comment]
correct @palash25

File: 2018.md (hunk 1 above)
Comment (rbitia) [reply 2018.md]: Yes!

**Comment by rbitia** [comment]
Oops I'll fix that. Thanks

**Pull request merged**